src/hooks/Guilds/contracts/useEncodedCall.ts

```diff
@@ -33,13 +33,13 @@ export const bulkEncodeCallsFromOptions = (options: Option[]): Option[] => {
         };
         if (!!decodedAction.approval) {
           const approvalCall = {
-            from: decodedAction.decodedCall.from,
-            to: decodedAction.approval?.token,
+            from: decodedAction.decodedCall.from, // Guild address
+            to: decodedAction.approval?.token, // Token address
             data: encodeAprovalCall(
-              decodedAction.decodedCall.from,
-              decodedAction.approval?.amount
+              decodedAction.decodedCall.to, // Spender: Contract we are doing the actual spending call to

```

Comment:
I think the spender is still being set incorrectly, looks like it is being set to the guild ID

![Screenshot 2022-08-19 at 10 15 48](https://user-images.githubusercontent.com/39137239/185587017-0f6fd564-c4b9-4b1b-8aaf-69b60f1ce79a.jpg)

Reply:
Weird. Let me test again.

Reply:
Fixed. Was only UI stuff displaying the wrong value, values on the call are ok!